server/server_test.go

```diff
@@ -703,23 +728,28 @@ func (cli *testServerClient) runTestLoadDataForListColumnPartition(t *testing.T)
 		dbt.MustExec("delete from t")
 		cli.prepareLoadDataFile(t, path, "1 a", "3 c", "4 e")
 		dbt.MustExec(fmt.Sprintf("load data local infile %q into table t", path))
+		rows.Close()
 		rows = dbt.MustQuery("select * from t order by id")
 		cli.checkRows(t, rows, "1 a", "3 c", "4 e")
 		// Test load data meet duplicate error.
 		cli.prepareLoadDataFile(t, path, "1 x", "2 b", "2 x", "7 a")
 		dbt.MustExec(fmt.Sprintf("load data local infile %q into table t", path))
+		rows.Close()
 		rows = dbt.MustQuery("show warnings")
 		cli.checkRows(t, rows,
 			"Warning 1062 Duplicate entry '1' for key 'idx'",
 			"Warning 1062 Duplicate entry '2' for key 'idx'")
+		rows.Close()
 		rows = dbt.MustQuery("select * from t order by id")
 		cli.checkRows(t, rows, "1 a", "2 b", "3 c", "4 e", "7 a")
 		// Test load data meet no partition warning.
 		cli.prepareLoadDataFile(t, path, "5 a", "100 x")
 		_, err := dbt.GetDB().Exec(fmt.Sprintf("load data local infile %q into table t", path))
 		require.NoError(t, err)
+		rows.Close()
 		rows = dbt.MustQuery("show warnings")
 		cli.checkRows(t, rows, "Warning 1526 Table has no partition for value from column_list")
+		rows.Close()
 		rows = dbt.MustQuery("select id,name from t order by id")
 		cli.checkRows(t, rows, "1 a", "2 b", "3 c", "4 e", "5 a", "7 a")
```

Comment:
Please add `rows.Close()` here.

Reply:
Although it is not outdated. it has been added rows.Close().